16_addressbook/main.js

```diff
@@ -0,0 +1,61 @@
+
+let myContacts = [];
+
+function handleAddContactClick() {
+    let firstName = document.getElementById('firstNameInput').value;
+    let lastName = document.getElementById('lastNameInput').value;
+    let emailAddress = document.getElementById('emailInput').value;
+
+    let newContact = {
+        firstName: firstName,
+        lastName: lastName,
+        emailAddress: emailAddress
+    };
+
+    myContacts.push(newContact);
+
+    saveContacts();
+    renderContacts();
+}
+
+
+function renderContacts() {
```

Comment:
I just finished telling Michelle that it makes more sense to declare the functions before calling them. I would put this, and saveContacts before the click handler